.github/workflows/system-tests-opensource.yml

```diff
@@ -162,19 +162,25 @@ jobs:
         # but this seems to work
         start args: '--addons=registry --insecure-registry="192.168.49.2:5000"'
 
-    - name: Get mlrun kit charts and create namespace
+    - name: Get mlrun ce charts and create namespace
       run: |
-        helm repo add v3io-stable https://v3io.github.io/helm-charts/stable
+        helm repo add mlrun-ce https://mlrun.github.io/ce
         helm repo update
         minikube kubectl -- create namespace ${NAMESPACE}
 
-    - name: Install MLRun Kit helm chart
+    - name: Install MLRun CE helm chart
       run: |
         helm --namespace ${NAMESPACE} \
-            install mlrun-kit \
+            install mlrun-ce \
             --debug \
             --wait \
             --timeout 600s \
+            -f https://raw.githubusercontent.com/mlrun/ce/development/charts/mlrun-ce/override-full.yaml \
+            --set kube-prometheus-stack.enabled=false \
+            --set mlrun.httpDB.dbType="sqlite" \
+            --set mlrun.httpDB.dirPath="/mlrun/db" \
+            --set mlrun.httpDB.dsn="sqlite:////mlrun/db/mlrun.db?check_same_thread=false" \
```

Comment:
Lets add a TODO comment to move to mysql, and move/copy the comment from the PR to the code